html/css/pattern-scaffolding.css

```diff
@@ -1,54 +0,0 @@
-/**
```

Comment:
Is there a reason why this file has been removed? I thought Pattern Lab needed it